naming/README.md

```diff
@@ -0,0 +1,25 @@
+# Solana Naming Service [WIP]
```

Comment:
I'm game to move this README.md to a `name-service.md` file in https://github.com/solana-labs/solana-program-library/tree/master/docs/src, so it shows up on the docs at spl.solana.com